spec/models/review_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe Review, type: :model do
+
+  describe "model validations" do
+    it "requires a value" do
+      empty_item = Review.new
+      expect(empty_item).to be_invalid
+      expect(empty_item.errors.keys).to include :rating
+    end
+  end
+  # does not work - fix!
```

Comment:
One really cool tool that you can use to make sure you come back to comments like this is by using ruby's quicktips: http://rubyquicktips.com/post/385665023/fixme-todo-and-optimize-code-comments.

Essentially, you write a comment you want to come back to as `# FIXME your comment` all throughout your project (as needed) and then in your project directory in your terminal run `$ rake notes` and it'll show you all these messages and where they are located in your project. (more detail in the link above)